Doofinder/Feed/Model/Data/SearchEngineOptionsStruct.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Doofinder\Feed\Model\Data;
+
+use JsonSerializable;
+
+class SearchEngineOptionsStruct  implements JsonSerializable
+{
+    private string $store_id;
```

Comment:
I would use the camelCase since it's the recommended standard here:

```suggestion
    private string $storeId;
```

+ PHPDocs :smile: